pre-install.xq

```diff
@@ -0,0 +1,18 @@
+xquery version "3.1";
+
+declare namespace xmldb="http://exist-db.org/xquery/xmldb";
+
+(: The following external variables are set by the repo:deploy function :)
+
+(: file path pointing to the exist installation directory :)
+declare variable $home external;
+(: path to the directory containing the unpacked .xar package :)
+declare variable $dir external;
+(: the target collection into which the app is deployed :)
+declare variable $target external;
+
+(: create public-repo-data and subcollections :)
+xmldb:create-collection("/db/apps", "public-repo-data"),
+for $col in ("icons", "metadata", "packages")
+return
+    xmldb:create-collection("/db/apps/public-repo-data", $col)
```

Comment:
Here is the missing newline

Reply:
@line-o Ok, I've fixed this and the other instances of this in the repository.